keyboards/massdrop/ctrl/keymaps/xanimos/README.md

```diff
@@ -0,0 +1,232 @@
+# Massdrop Ctrl Xanimos
```

Comment:
please rename this file to be lower case, and please remove the html entities.

Reply:
Resolved in d475857 and 9a96897